lib/filterx/object-datetime.c

```diff
@@ -200,6 +200,64 @@ _repr(FilterXObject *s, GString *repr)
   return TRUE;
 }
 
+FilterXObject *
+filterx_datetime_strptime(GPtrArray *args)
+{
+  if (args == NULL || args->len < 2)
+    {
+      msg_error("FilterX: Failed to create datetime object: invalid number of arguments. "
```

Comment:
we need to get going with a better error handling in FilterX expressions. These scattered log messages are not going to cut it.